src/dune_pkg/opam_repo.ml

```diff
@@ -360,10 +360,13 @@ let load_all_versions ts opam_package_name =
     if OpamPackage.Version.Map.mem version acc
     then acc
     else OpamPackage.Version.Map.add version (repo, package) acc)
-  >>| OpamPackage.Version.Map.values
-  >>= Fiber.parallel_map ~f:(fun (repo, pkg) -> load_opam_package repo pkg)
+  >>| OpamPackage.Version.Map.to_seq
```

Comment:
Could you explain why you are using seq here?

Reply:
Because there's no `Map.to_list` function.